Modules/_json.c

```diff
@@ -1356,6 +1356,31 @@ _steal_accumulate(_PyUnicodeWriter *writer, PyObject *stolen)
     return rval;
 }
 
+// TODO: this method could be cached
+PyObject* _create_newline_indent(PyObject* indent, Py_ssize_t indent_level)
+{
+    PyObject* start = PyUnicode_FromString("\n");
```

Comment:
Don't we need to initialise `newline_indent` with `NULL`?

```suggestion
    PyObject* start = PyUnicode_FromString("\n");
    PyObject* newline_indent = NULL;
```